README.md

```diff
@@ -24,6 +24,7 @@ Pending code reviews represent blocked threads of execution. To prevent getting
 ## Quality
 
 - prioritize quality over speed | [the only way to go fast is to go well](http://butunclebob.com/ArticleS.UncleBob.VehementMediocrity)
+- [test, automate, test, automate](docs/automate-tests.md)
```

Comment:
Might be a good time to mention using linting tools or other kinds of automated robot-tasks that make it easier to review PRs?

For example, the `usm-apis` repo runs a couple of easy-automated formatters that keep the code clean. These can run as part of the commit process itself if desired. They also run a part of the CI/CD checks upon PR submission and must pass before PR merge.

https://github.com/unionstreetmedia/usm-apis/blob/main/CONTRIBUTING.md#code-style

Reply:
I agree. Ok if I merge this PR and we can add linting separately?